### PR TITLE
test: Use couchdb for bdd test

### LINKED ISSUE #2160: Use couchdb for bdd test

### PR DESCRIPTION
**Pull request by fqutishat**
closes #2160

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/didcomm/packer/authcrypt/pack.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 const (
 	encodingType = "didcomm-envelope-enc"
 	// ThirdPartyKeysDB is a store name containing keys of third party agents.
-	ThirdPartyKeysDB = "thirdPartyKeysDB"
+	ThirdPartyKeysDB = "thirdpartykeysdb"
```
Comment (llorllale) [inline pkg/didcomm/packer/authcrypt/pack.go:36]: what's the reason for this change?

Comment (fqutishat) [reply pkg/didcomm/packer/authcrypt/pack.go:36]: couchdb will not accept upper case

Comment (llorllale) [reply pkg/didcomm/packer/authcrypt/pack.go:36]: You are right: https://docs.couchdb.org/en/stable/api/database/common.html#put--db.

Patching these sorts of db-specific bugs everywhere is untenable.

**Review by llorllale — APPROVED** [review]

File: test/bdd/fixtures/agent-rest/.env [hunk 2]
```diff
@@ -99,3 +99,7 @@ DAVE_ROUTER_WEBHOOK_PORT=10074
 
 
 MYSQL_IMAGE_TAG=8.0.20
+
+COUCHDB_IMAGE=couchdb
+COUCHDB_IMAGE_TAG=2.3.1
```
Comment (rolsonquadras) [inline test/bdd/fixtures/agent-rest/.env:104]: Can we use the latest version ie v3.1 ? https://hub.docker.com/_/couchdb

**Review by rolsonquadras — APPROVED** [review]

File: pkg/didcomm/packer/authcrypt/pack.go (hunk 1 above)
Comment (Baha-sk) [reply pkg/didcomm/packer/authcrypt/pack.go:36]: weird indeed, but I'm ok with all lower case name for this store used specifically for the new Authcrypt packer.

**Review by Baha-sk — APPROVED** [review]

File: test/bdd/fixtures/agent-rest/.env (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/agent-rest/.env]: done
